### ISSUE TITLE
Schema are loaded strictly based from the current directory

### ISSUE THREAD

**Issue by alexander-bauer** [vesposito/easyucs#6]
This affects the current master of this repository, as well as my pull request (#3). Using `config push` against a UCSM instance triggers a function that tries to load `schema/ucs/ucsm/master.json` relative to the current directory. This means that it must be run from the repository root (or the package root, in my branch).

The [relevant line](https://github.com/vesposito/easyucs/blob/master/config/manager.py#L1108) is
```
        json_file = open("schema/ucs/ucsm/master.json")
```

@vesposito is it reasonable to assume that schemas could be packaged with EasyUCS and not change without package updates, or is it necessary to support that users could retrieve schemas and store them somewhere that EasyUCS would subsequently use? Would it be sensible to add an optional `--schema` flag that is a path to a "schema root", that defaults to the one distributed with EasyUCS?

**Comment by vesposito** [comment]
Yes, we can assume that schemas are packaged with EasyUCS and not change without package updates. They are intimately linked to the features EasyUCS support, so being able to modify/update them without modifying the EasyUCS engine would not make much sense IMHO.
So adding an optional `--schema` flag should not be very useful and would complicate things for regular users.

I think loading the schemas from a relative path to the `EASYUCS_ROOT` is a good idea and should be a good fix for this issue.

Maybe this issue should be fixed in a specific pull request, separate from your "pip package management" modifications?
